src/GeneratorV2/Writing/Writer.cs

```diff
@@ -165,7 +166,7 @@ private static void WriteNativeMethod(NativeFunction function, bool postfixName,
             writer.WriteLine("using System.Runtime.InteropServices;");
             writer.WriteLine("using OpenTK.Mathematics;");
             // FIXME: Figure out whether we wanna use opentk's half or system.numerics. Does it even make a difference?
-            writer.WriteLine("using Half = System.Half;");
+            writer.WriteLine("using Half = OpenTK.Mathematics.Half;");
```

Comment:
We want to move to `System.Half` but I think that's for another PR.

Reply:
Yeah, we can switch this at any time. But that requires moving the vector class in opentk.mathemathics over to using System.Half too.